### ISSUE TITLE
Example in docs doesn't seem to work as expected. 

### ISSUE THREAD

**Issue by TwistingTwists** [am-kantox/finitomata#59]
When trying to run the example given at, https://hexdocs.pm/finitomata/readme.html#example

These are iex outputs

```
iex> Finitomata.start_fsm MyFSM, "My first FSM", %{foo: :bar}
{:ok, #PID<0.888.0>}
iex(6)> [warning] [⚐ ↹] transition not exists or not allowed (:responds)
[warning] [✗ ↹] [state: #Finitomata<[name: "My first FSM", state: [current: :*, previous: nil, payload: %{foo: :bar}], internals: [errored?: false, persisted?: false, timer: false]]>, event: :__start__, event_payload: %{__retries__: 1, payload: nil}]
```

and the following examples don't work as expected. 

```
iex> Finitomata.transition "My first FSM", {:to_s2, nil}
[warning] [⚐ ↹] transition not exists or not allowed (:responds)
:ok
[warning] [✗ ↹] [state: #Finitomata<[name: "My first FSM", state: [current: :*, previous: nil, payload: %{foo: :bar}], internals: [errored?: false, persisted?: false, timer: false]]>, event: :to_s2, event_payload: nil]

```

```
iex(8)> Finitomata.state "My first FSM"
#Finitomata<[
  name: "My first FSM",
  state: [current: :*, previous: nil, payload: %{foo: :bar}],
  internals: [errored?: false, persisted?: false, timer: false]
]>

```

**Comment by am-kantox** [comment]
Hi @TwistingTwists, thanks for reporting, I updated the documentation accordingly to the latest changes, fixed some minors, and incorporated your fixes.

> Package published to https://hex.pm/packages/finitomata/0.14.4
> (f267569f69ddf1bae11fde1dbeee3808158d149589e355571f3c90b40a40b69a)

Now it should be inline with docs :)

**Comment by am-kantox** [comment]
Fixed in dc08dd64d96b301cb1899fe44c571e8438390006.
